agent/consul/acl_endpoint.go

```diff
@@ -2383,3 +2383,17 @@ func (a *ACL) Authorize(args *structs.RemoteACLAuthorizationRequest, reply *[]st
 	*reply = responses
 	return nil
 }
+
+// ResolveIdentityFromToken passes through a request from the ACL type to the ACL's srv delegate.
+func (a *ACL) ResolveIdentityFromToken(args *structs.ACLRequest, reply *structs.ACLIdentity) error {
```

Comment:
I dont think this new RPC endpoint should be necessary. Anywhere you use this should be be able to use the `ACL.TokenRead` endpoint. That will return a `structs.ACLToken` which impelements the `ACLIdentity` interface.

Reply:
Ah! Good shout -- will do

Reply:
Cut `agent/consul/acl_endpoint.go (a *ACL) ResolveIdentityFromToken ` in latest revision